### PR TITLE
github action for pytest

### PR DESCRIPTION
**Pull request by ErnestaP**

### CODE REVIEW

File: .github/workflows/pytest.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Install Python 2.7.16 
+        uses: actions/setup-python@v3
+        with:
+          python-version:  2.7.16 
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt 
+      - name: Run tests with pytest and generate report
+        run: coverage run -m pytest
```
Comment (drjova) [inline .github/workflows/pytest.yaml:24]: I'm curious 😅

File: .github/workflows/pytest.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Upload Coverage to Codecov
+        uses: codecov/codecov-action@v2
+
```
Comment (drjova) [inline .github/workflows/pytest.yaml:27]: the empty space here 😄

File: .github/workflows/pytest.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Python 2.7.16 
+        uses: actions/setup-python@v3
+        with:
+          python-version:  2.7.16 
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
```
Comment (drjova) [inline .github/workflows/pytest.yaml:22]:
```suggestion
          pip install -r requirements.txt -e .[tests,docs]
```

File: .github/workflows/pytest.yaml (hunk 2 above)
Comment (drjova) [reply .github/workflows/pytest.yaml:27]: you can configure your editor to automatically do that

File: .github/workflows/pytest.yaml (hunk 1 above)
Comment (ErnestaP) [reply .github/workflows/pytest.yaml:24]: About what? 😄

**Review by drjova — APPROVED** [review]

**Pull request merged**